modules/cnf-cpu-infra-container.adoc

```diff
@@ -5,17 +5,55 @@
 [id="cnf-cpu-infra-container_{context}"]
 = Restricting CPUs for infra and application containers
 
-You can reserve cores (threads) from a single NUMA node for operating system housekeeping tasks and put your workloads on another NUMA node. Partitioning the CPUs this way can prevent the housekeeping processes from impacting latency-sensitive application processes. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. 
+Housekeeping and workloads may use the central processing units (CPUs) in a way that would impact latency sensitive processes running on those CPUs. Partitioning the CPUs prevents the processes from interfering with each other. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. Use the following table see where process' CPUs run after tuning the node using the Performance Addon Operator:
 
-You can ensure that housekeeping tasks and workloads run on separate NUMA nodes by specifying two groups of CPUs in the `spec` section of the performance profile. 
+.Process' CPU assignments
+[%header,cols=2*] 
+|===
+|Process type
+|Details
+
+|Burstable and best-effort pods
+|Runs on any CPU not used by a low latency container, including reserved
+
+|Infrastructure pods
+|Runs on any CPU not used by a low latency container, including reserved
+
+|Interrupts
+|Redirected to reserved CPUs (optional in {product-version} and later)
+
+|Kernel processes
+|Pinned to reserved CPUs
+
+|Latency sensitive workload pods
+|Pinned to a specific set of exclusive CPUs from the isolated pool
+
+|OS processes/systemd services
+|Pinned to reserved CPUs
+|===
+
+The exact partitioning pattern to use depends on many factors like hardware, workload characteristics and the expected system load. Some sample use cases are as follows:
+
+* If the latency sensitive workload uses specific hardware (like network interface cards (NICs)), ensure that the CPUs in the isolated pool are as "close" to this hardware as possible. Place the workload in at least the same Non-Uniform Memory Access (NUMA) node.
+
+* The reserved pool is used for handling all interrupts. When depending on system networking, allocate a sufficiently sized reserve pool to handle all the incoming packet interrupts. In {product-version} and later, workloads can optionally be labeled as sensitive. For more information, see the Red Hat Knowledgebase solution xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus[Managing device interrupt processing for guaranteed pod isolated CPUs].
```

Comment:
```suggestion
* The reserved pool is used for handling all interrupts. When depending on system networking, allocate a sufficiently sized reserve pool to handle all the incoming packet interrupts. In {product-version} and later, workloads can optionally be labeled as sensitive. For more information, see the xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus[Managing device interrupt processing for guaranteed pod isolated CPUs] section of this document.
```

Reply:
It renders to an internal link within the same doc, also this is the documentation, not knowledge base.

Reply:
Good catch! Updated the link to reflect the correct location.